changelog/fix-2199-woopay-duplicate-payments

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: fix
+Comment: Fix WooPay duplicate charges.
+
+
```

Comment:
Nit: Remove the additional empty line.

Reply:
This file is automatically generated and will be removed in the release process, so that's fine 🙂